docs/Anjay_integrations/ESP32_integration/M5Stick.md

```diff
@@ -0,0 +1,132 @@
+- A user with access to the Coiote IoT Device Management platform.
+
+## Step 1: Clone Anjay ESP32 client repository
+
+Enter the command line interface on your machine and paste the following command:
+
+   ```
+   git clone --recursive https://github.com/AVSystem/Anjay-esp32-client
+   ```
+
+## Step 2: Configure the client using an NVS partition
+
+0. Create a `nvs_config.csv` file and provide your credentials in [wifi_ssid], [wifi_password], [identity], [psk], [lwm2m_server_uri] (without the `[]` brackets). You can use the following snippet as a template:
+
+    !!! important
+        The **Identity** parameter stands for both the device endpoint name and its PSK identity, therefore these two must be identical in Coiote DM.  
+
+    ```
+    key,type,encoding,value
+    config,namespace,,
+    wifi_ssid,data,string,[wifi_ssid]
+    wifi_pswd,data,string,[wifi_password]
+    wifi_inter_en,data,u8,1
+    identity,data,string,[identity]
+    psk,data,string,[psk]
+    uri,data,string,[lwm2m_server_uri]
+    writable_wifi,namespace,,
+    wifi_ssid,data,string,[wifi_ssid]
+    wifi_pswd,data,string,[wifi_password]
+    wifi_inter_en,data,u8,0
+    ```
+
+    !!! Note
+        The additional parameters under the **writable_wifi** namespace are used to provide a secondary Wi-Fi configuration (it is not obligatory). This allows for switching between Wi-Fi configurations while the device is running.
+
+0. Generate the NVS partition:
+
+```
+python3 tools/nvs_partition_gen.py generate nvs_config.csv nvs_config.bin 0x4000
+```
+
+![Client configuration](images/nvs_config.png "Client configuration"){: style="float: left;margin-right: 1177px;margin-top: 17px;"}
+
+## Step 4: Add device to Coiote DM
+
+To connect your M5StickC to the Coiote IoT Device Management LwM2M Server, use your access to a Coiote DM installation, or register at https://www.avsystem.com/try-anjay/ to get access.
+
+To connect the board:
+
+0. Log in to Coiote DM and from the left side menu, select **Device Inventory**.
+0. In **Device Inventory**, click **Add device**.
+0. Select the **Connect your LwM2M device directly via the Management server** tile.
+![Add via Mgmt](images/mgmt_tile.png "Add via Mgmt")
+0. In the **Device credentials** step:
+    - In the **Device ID** field, type the name provided in the `nvs_config.csv`, e.g. `ESP32_test`.
+      ![Device credentials step](images/add_mgmt_quick.png "Device credentials step")
+    - In the **Security mode** section, select the **PSK** mode.
+    - In the **Key identity** field, type the name provided in the `nvs_config.csv`, e.g. `ESP32_test`.
+    - In the **Key** field, type the `psk` key provided in the `nvs_config.csv`.  
+0. Click the **Add device** button and **Confirm** in the confirmation pop-up.
+0. In the **Connect your device** step, wait for the board to connect.
```

Comment:
`wait for the board to connect`
If we should wait here for the connection to the board, then maybe `Flash the board and run device` step should be presented first?

Reply:
If so probably this image also should be move here 
![image](https://user-images.githubusercontent.com/17883057/167643138-4d1d151c-35d0-46f8-a839-7b8043e16b15.png)

Reply:
i changed the wording a bit to resolve this in a different way